index.js

```diff
@@ -75,13 +66,13 @@ app.get('/thanks', (req, res) => {
 })
 
 app.get('/live_poll/:id', (req, res) => {
-  const url = urlGen(req)
+  const url = new Helper(req).urlGen
```

Comment:
Yea, now that I've seen this in action—I definitely don't love it.

The line you deleted was better.